src/version.h

```diff
@@ -1,4 +1,5 @@
 #define SERVER_NAME "valkey"
+#define SERVER_TITLE "Valkey"
```

Comment:
Was the reason to make this SERVER_TITLE to make it independent from SERVER_NAME so we could add the compatibility switch later?

Reply:
It's "Valkey" vs "valkey". It was discussed elsewhere (some clicks away).

@0del please update the PR title and write a description about the change, including what SERVER_TITLE is.

Reply:
And yes, also for the compat switch.

Reply:
Maybe call it SERVER_NAME_LOWER then? At first glance I didn't notice the difference

Reply:
TITLE is in "title case", i.e. first letter capitalized.

(It was my suggestion.)